### ISSUE TITLE
Importing of screen files are required now ?

### ISSUE THREAD

**Issue by akarabach** [Milad-Akarie/auto_route_library#1432]
`import 'package:auto_route/auto_route.dart';
import 'package:flutter/cupertino.dart';
import 'package:flutter/material.dart';

**import '../screens/home_screen.dart';** <----- **_without this import I'have a problem_ Error (Xcode): lib/router/app_router.gr.dart:20:22: Error: Couldn't find constructor 'HomeScreen'.** 

part 'app_router.gr.dart';

@AutoRouterConfig(replaceInRouteName: 'Screen|Page,Route')
class AppRouter extends _$AppRouter {
  @override
  final List<AutoRoute> routes = [
    AutoRoute(
      path: '/',
      page: HomeRoute.page,
   ),
    RedirectRoute(path: '*', redirectTo: '/'),
  ];
}
`

`// GENERATED CODE - DO NOT MODIFY BY HAND

// **************************************************************************
// AutoRouterGenerator
// **************************************************************************

// ignore_for_file: type=lint
// coverage:ignore-file

part of 'app_router.dart';

abstract class _$AppRouter extends RootStackRouter {
  _$AppRouter([GlobalKey<NavigatorState>? navigatorKey]) : super(navigatorKey);

  @override
  final Map<String, PageFactory> pagesMap = {
    HomeRoute.name: (routeData) {
      return AutoRoutePage<dynamic>(
        routeData: routeData,
        child: const HomeScreen(),
      );
    }
  };
}

/// generated route for
/// [HomeScreen]
class HomeRoute extends PageRouteInfo<void> {
  const HomeRoute({List<PageRouteInfo>? children})
      : super(
          HomeRoute.name,
          initialChildren: children,
        );

  static const String name = 'HomeRoute';

  static const PageInfo<void> page = PageInfo<void>(name);
}
`

auto_route: ^6.0.1
auto_route_generator: ^6.0.0

**Comment by Milad-Akarie** [comment]
@akarabach do you have to use the part file approach?

**Comment by akarabach** [comment]
no, it's just more familiar for me since previous versions and looks like all examples are using this approach
